### PR TITLE
Topics/release to public

### PR DESCRIPTION
**Pull request by sashless**
* [x] add test setup
* [x] add dev setup
* [x] add readme
* [x] move code
* [x] make it work without jquery

in preparation for https://github.com/easybib/issues/issues/4698

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "format-preserver",
+  "version": "0.0.1",
+  "description": "Used to preserve formats like BUI when stripping markup and styles from HTML string",
+  "main": "src/format-preserver/format-preserver.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (rwos) [inline package.json]: can probably go, no? (or put karma in here)

Comment (till) [inline package.json]: :+1: put karma there and use `npm test` in the `Makefile` and remove the export for `node_modules/.bin`

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+language: node_js
+
+node_js: 4.2.1
```
Comment (till) [inline .travis.yml]: Is this a nodejs **stable** release?

File: package.json (hunk 1 above)
Comment (till) [inline package.json]: And gulp, etc. as well.

File: .travis.yml (hunk 2 above)
Comment (sashless) [inline .travis.yml]: according to https://nodejs.org/en/blog/release/v4.2.1/ it is LTS , that means yes

File: package.json (hunk 1 above)
Comment (sashless) [inline package.json]: good thing :+1:

**Comment by sashless** [comment]
addressed all the comments, want to have a second look @rwos , otherwise i just merge ?

**Comment by rwos** [comment]
@sashless :eyes:

**Comment by rwos** [comment]
@sashless looks good to me! :+1:

**Pull request merged**